### PR TITLE
Link to gh-profile on message gravatar.

### PR DESCRIPTION
**Pull request by hemanth**
Had to use `shell`

### CODE REVIEW

File: lib/elements/messages.js [hunk 1]
```diff
@@ -42,7 +42,11 @@ Messages.prototype.render = function (channel, users) {
       var verified = !/Anonymous/i.test(msg.username) && msg.valid
       var userUrl = verified ? 'http://github.com/' + msg.username : '#'
       return h('li.message.clearfix', [
-        h('img.avatar', { src: msg.avatar }),
+        h('img.avatar', { src: msg.avatar , style: { cursor: 'pointer' },
+          onclick: function (e) {
+            if (userUrl !== '#')
+            shell.openExternal(userUrl)
+          }),
```
Comment (shama) [inline lib/elements/messages.js]: I think you're missing a closing `}` here.

**Comment by shama** [comment]
Thanks for the PR! Another way to accomplish this would be to wrap in an `<a/>` tag, like:

```js
h('a', { href: userUrl, target: '_blank' }, h('img.avatar', { src: msg.avatar }))
```

**Comment by hemanth** [comment]
@shama Tried that first, but the `a` tag does not seem to open in a new window/browser even with `target: '_blank'`

File: lib/elements/messages.js (hunk 1 above)
Comment (hemanth) [inline lib/elements/messages.js]: Might be, will recheck, sent this PR via mobile device heh heh.

**Comment by shama** [comment]
@hemanth Ah strange, I thought that worked. Instead of using `shell.openExternal` here (as that ties this element to the desktop app only), do this instead:

```js
onclick: function (e) {
  self.send('openUrl', userUrl)
}
```

which should emit the event up until the `desktop.js` listener catches it to call `shell.openExternal`: https://github.com/moose-team/friends/blob/master/desktop.js#L29

File: lib/elements/messages.js (hunk 1 above)
Comment (hemanth) [inline lib/elements/messages.js]: It's balanced.

**Comment by hemanth** [comment]
@shama Neat, thanks, makes much more sense now, have updated it to avoid direct call via shell.

**Comment by shama** [comment]
Thanks!

**Comment by hemanth** [comment]
:+1:

**Pull request merged**